### DISCUSSION TITLE
Showing informational content in a card

### DISCUSSION THREAD

**Discussion by mindytom** [twilio-labs/paste#2303] [Q&A] [answered]
Hi DSYS, 

I see on the Paste website y'all use this component that is similar to an alert but different. 
<img width="669" alt="Screen Shot 2022-03-18 at 2 32 37 PM" src="https://user-images.githubusercontent.com/65037539/159086112-f0701473-5476-4073-aa58-9dcaff08b648.png">

I'd like to use something similar for [Link Shortening](https://www.figma.com/file/QKL8U8ppyYS9l749sLGVrH/Link-Shortening?node-id=1110%3A151663). The card previews whatever link a user selects from the dropdown combobox in that format. From usability studies customers have responded well and have not expressed any confusion. Do you have any suggestions on how we should implement this? 
<img width="831" alt="Screen Shot 2022-03-18 at 2 38 33 PM" src="https://user-images.githubusercontent.com/65037539/159086275-7f19edd2-4947-4be1-b603-7b993b021fd5.png">

**Comment by nkrantz** [comment, marked as answer]
Hi @mindytom - this is a good use case for a Callout component. Although it's not yet available as part of the system, Loreina started [building out the Callout](https://www.figma.com/file/GfY11A2VW638eFK48nDtdE/Asset-Documentation?node-id=1220%3A40) we use on our docs site (in your first screenshot) in Figma, so feel free to use that. For code, your team can copy what we have on the site as well ([here's how we've built it](https://github.com/twilio-labs/paste/blob/main/packages/paste-website/src/components/callout/index.tsx)). Hope this helps!

**Reply by mindytom** [reply]
@jeevan-sg Tagging here for visibility.
